src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionDescriptorBuilder.cs

```diff
@@ -727,8 +727,16 @@ private static bool IsAttributeRoutedAction(ControllerActionDescriptor actionDes
             {
                 var routeTemplate = action.AttributeRouteInfo?.Template ?? nullTemplate;
 
-                var verbs = action.ActionConstraints.OfType<HttpMethodActionConstraint>().FirstOrDefault()?.HttpMethods;
-                var formattedVerbs = string.Join(", ", verbs.OrderBy(v => v, StringComparer.Ordinal));
+                var verbs = action.ActionConstraints?
+                    .OfType<HttpMethodActionConstraint>()
```

Comment:
Curious - what would you do instead? We often like the `.` to start the next line, so what do you do here?